plasmapy/formulary/parameters.py

```diff
@@ -694,6 +702,15 @@ def Hall_parameter(n,
     Returns
     -------
     astropy.units.quantity.Quantity
+
+    Examples
+    --------
+    >>> from astropy import units as u
+    >>> Hall_parameter(1e10*u.m**-3,2.8e3*u.eV,2.3*u.T,'He-4 +1')
+    <Quantity 7.26446755e+16>
+    >>> Hall_parameter(1e10*u.m**-3,5.8e3*u.eV,2.3*u.T,'He-4 +1')
```

Comment:
With dense argument lists like this, some spaces go a long way in making this kind of thing more readable:
```suggestion
     >>> Hall_parameter(1e10 * u.m**-3, 2.8e3 * u.eV, 2.3 * u.T, 'He-4 +1') 
    <Quantity 7.26446755e+16>
    >>> Hall_parameter(1e10 * u.m**-3, 5.8e3 * u.eV, 2.3 * u.T, 'He-4 +1')
```